e2e/Tests/prerequisites/test-resources.json

```diff
@@ -352,18 +343,11 @@
     },
     "workspaceId": {
       "type": "string",
-      "value": "[if(parameters('EnableIotHubSecuritySolution'), reference(resourceId('Microsoft.OperationalInsights/workspaces', parameters('OperationalInsightsName')), '2017-03-15-preview').customerId, '')]"
+      "value": "[if(parameters('EnableIotHubSecuritySolution'), format('{0}', reference(resourceId('Microsoft.OperationalInsights/workspaces', parameters('OperationalInsightsName')), '2017-03-15-preview').customerId), '')]"
     },
     "keyVaultName": {
       "type": "string",
       "value": "[parameters('KeyVaultName')]"
     }
-  },
-  "metadata": {
-    "_generator": {
-      "name": "bicep",
-      "version": "0.3.1.62928",
-      "templateHash": "129270437041242196"
-    }
   }
-}
+}
```

Comment:
extra line at EOF?

Reply:
This is a generated file. We don't hand edit it.